lib/application_insights/rack/track_request.rb

```diff
@@ -54,7 +58,11 @@ def call(env)
           :url => request.url
         }
 
-        @client.track_request id, start_time, duration, status, success, options
+        # Setup operation context
+        @client.context.operation.id = operation_id(request_id)
```

Comment:
Is ```@client``` local to each request? This is my major concern of using the Context from ```@client```.

Reply:
I'll update my implementation now to call `#clone` on the client before setting the values